tests/protractor/e2e/doc_by_place.js

```diff
@@ -0,0 +1,196 @@
+var utils = require('../utils'),
+    // auth = require('../auth'),
+    async = require('async');
+
+describe('Filter doc_by_place', function() {
+
+  'use strict';
+
+  var documentsToReturn = [
```

Comment:
Use uppercase for constants : https://github.com/medic/medic-docs/blob/master/md/dev/style-guide.md#use-uppercase-for-constants
(bam I quoted the style guide)